newIDE/app/src/UI/Notification/NotificationListItem.js

```diff
@@ -0,0 +1,165 @@
+// @flow
+
+import * as React from 'react';
+import { I18n } from '@lingui/react';
+import { Trans } from '@lingui/macro';
+import { type Notification } from '../../Utils/GDevelopServices/Notification';
+import { ListItem } from '../List';
+import { getRelativeOrAbsoluteDisplayDate } from '../../Utils/DateDisplay';
+import CoinOutline from '../CustomSvgIcons/CoinOutline';
+import Annotation from '../CustomSvgIcons/Annotation';
+import Cart from '../CustomSvgIcons/Cart';
+import { shortenString } from '../../Utils/StringHelpers';
+import GDevelopThemeContext from '../Theme/GDevelopThemeContext';
+import RouterContext, {
+  type RouteArguments,
+} from '../../MainFrame/RouterContext';
+
+const notificationTypeToIcon = {
+  'credits-drop': <CoinOutline />,
+  'one-game-feedback-received': <Annotation />,
+  'multiple-game-feedback-received': <Annotation />,
+  'claimable-asset-pack': <Cart />,
+};
+
+const getNotificationPrimaryTextByType = (
+  notification: Notification
+): React.Node => {
+  if (
+    notification.type === 'credits-drop' &&
+    notification.data.reason.startsWith('subscription')
+  ) {
+    return (
+      <Trans>
+        You received {notification.data.amount} credits thanks to your
+        subscription
+      </Trans>
+    );
+  }
+  if (notification.type === 'one-game-feedback-received') {
+    if (notification.data.playerName) {
+      // Prevent prettier formatting that puts the first double quote at the end of the
+      // previous line, adding a white space between the double quote and the comment.
+      // prettier-ignore
```

Comment:
to do this, I managed to create an intermediary prop like 
```
const gameNameAndShortenFeedback = `${gameName}: ${shortString}`;
```

Then use it in the Trans

Reply:
Oh yeah that works afterwards when the macro extracts the sentences to translate?

Reply:
Remember though that this will create a different translation, and many language might:
- use different spacing betweent the colom
- not use a colon at all
- could even reorder words differently